src/Components/Toast/Toast.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { toast, ToastContainer } from "react-toastify";
+
+export const ToastLogIn = ({condition}) => {
+  let cond = condition;
+
+  return (
+    <ToastContainer limit={1}>
+        
+      {!cond
+        ? toast.error("Такого акаунту немає aбо ви не заповнили поля!", {
```

Comment:
message for a toast shoud be somewhere as `const` and then u reuse it. Because that message could be reused